### PR TITLE
The invoke command

### PR DESCRIPTION
**Pull request by timqian**
sls invoke: https://app.asana.com/0/1200011502754281/1200211330092197

sls invoke local: https://app.asana.com/0/1200011502754281/1200276428733645

### CODE REVIEW

File: src/cli/commands-cn/invoke/index.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      throw new Error('找不到指定的路径文件, 请检查后重试');
+    }
+  }
+
+  if (dataValue && !isJson(dataValue)) {
+    throw new Error('传入的 data 不是序列化 JSON, 请检查后重试');
+  }
+
+  const instanceDir = process.cwd();
+  await utils.checkBasicConfigValidation(instanceDir);
```
Comment (zongUMR) [inline src/cli/commands-cn/invoke/index.js:45]: 这一步需要放到函数开始，local invoke 也需要这个校验

**Review by zongUMR — DISMISSED** [review]

File: src/cli/commands-cn/invoke/invoke-local/sls-egg.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+'use strict';
```
Comment (timqian) [inline src/cli/commands-cn/invoke/invoke-local/sls-egg.js:1]: 这个文件不需要的话是否应该删掉？

Comment (zongUMR) [reply src/cli/commands-cn/invoke/invoke-local/sls-egg.js:1]: 先不删掉，这个框架的需求在以后的迭代中可能还要用

**Review by zongUMR — DISMISSED** [review]

**Review by zongUMR — APPROVED** [review]

**Review by zongUMR — APPROVED** [review]

**Review by zongUMR — APPROVED** [review]

**Comment by G-Rath** [comment]
@timqian @zongUMR `jsome` looks to be abandoned, and uses a vulnerable version of `yargs` (which is also not required in this context, since it's not being used as a cli) - would you be open to finding a replacement?

Ideally it'd be great if you'd be happy to just use the `inspect` function from `utils` as that ships with Node so has no external dependencies, even if just as a quick replacement.

I'm happy to make PRs if you're open to this :)

**Comment by zongUMR** [comment]
> @timqian @zongUMR `jsome` looks to be abandoned, and uses a vulnerable version of `yargs` (which is also not required in this context, since it's not being used as a cli) - would you be open to finding a replacement?
> 
> Ideally it'd be great if you'd be happy to just use the `inspect` function from `utils` as that ships with Node so has no external dependencies, even if just as a quick replacement.
> 
> I'm happy to make PRs if you're open to this :)

Hi Gareth, thanks for your information. Looks like the `jsome` is not a good package as you said it is abandoned, welcome any PR for this feature. :)

**Comment by G-Rath** [comment]
@zongUMR awesome, and thanks for the quick response!

I've created #955 to replace `jsome` with `util.inspect` :)

**Pull request merged**